lib/NetPacks.h

```diff
@@ -571,8 +533,9 @@ struct ChangeFormation : public CPackForClient
 
 struct RemoveObject : public CPackForClient
 {
-	RemoveObject(){}
-	RemoveObject(ObjectInstanceID ID){id = ID;};
+	RemoveObject() = default;
+	RemoveObject(const ObjectInstanceID & ID)
+		: id(ID){};

```

Comment:
Another weird formatting. `{}` should be on separate line and without semicolon at the end.
And I see similar form in code below as well.

Reply:
Semicolon was there initially

Reply:
formatting issues resolved AFAIK.

Reply:
Yes, semicolon was here before, but it is unnecessary & apparently messed up formatting for some reason. I thought tools like clang-format should remove it.

Anyway, looks to be OK now.